src/train_new.py

```diff
@@ -8,6 +8,7 @@
 import torch
 import torch.nn.functional as F
 import torch.optim as optim
+from torch.utils.tensorboard import SummaryWriter

```

Comment:
torch 0.4.0 does not include tensorboard. Please use tensorboardX instead:
```
from tensorboardX import SummaryWriter
```

Reply:
Done